dynamodb/src/test/java/docs/javadsl/ExampleTest.java

```diff
@@ -59,41 +66,26 @@ public void checkForStageLeaks() {
     StreamTestKit.assertAllStagesStopped(materializer);
   }
 
-  @Test
-  public void setCredentialsProvider() {
-    // #credentials-provider
-    AWSCredentialsProvider provider = // ...
-        // #credentials-provider
-        new AWSStaticCredentialsProvider(new BasicAWSCredentials("accessKey", "secretKey"));
-    // #credentials-provider
-    DynamoSettings settings =
-        DynamoSettings.create("eu-west-1", "localhost").withCredentialsProvider(provider);
-    // #credentials-provider
-    assertEquals("eu-west-1", settings.region());
-  }
-
   @Test
   public void listTables() throws Exception {
+    // format: off
     // #simple-request
-    final Source<ListTablesResult, NotUsed> listTables =
-        DynamoDb.listTables(new ListTablesRequest());
+    final CompletionStage<ListTablesResponse> listTables =
+        DynamoDb.single(
+            client, AwsOp.listTables(), ListTablesRequest.builder().build(), materializer);
     // #simple-request
-    ListTablesResult result =
-        listTables
-            .runWith(Sink.head(), materializer)
-            .toCompletableFuture()
-            .get(5, TimeUnit.SECONDS);
-    assertNotNull(result.getTableNames());
+    // format: on
+    ListTablesResponse result = listTables.toCompletableFuture().get(5, TimeUnit.SECONDS);
+    assertNotNull(result.tableNames());
   }
 
   @Test
   public void allowMultipleRequests() throws Exception {
     // #flow
     Source<String, NotUsed> tableArnSource =
-        Source.single(AwsOp.create(new CreateTableRequest().withTableName("testTable")))
-            .via(DynamoDb.flow())
-            .map(r -> (CreateTableResult) r)
-            .map(result -> result.getTableDescription().getTableArn());
+        Source.single(CreateTableRequest.builder().tableName("testTable").build())
+            .via(DynamoDb.flow(client, AwsOp.createTable(), 1))
+            .map(result -> result.tableDescription().tableArn());
```

Comment:
This should be expanded to actually demonstrated multiple requests, according to the test name and as in Scala spec:

```
  .map(result -> DescribeTableRequest.builder().tableName(result.tableDescription().tableName()).build())
  .via(DynamoDb.flow(client, AwsOp.describeTable(), 1))
```